src/control/server/drpc.go

```diff
@@ -104,9 +107,10 @@ func newDrpcCall(module int32, method int32, bodyMessage proto.Message) (*drpc.C
 	}
 
 	return &drpc.Call{
-		Module: module,
-		Method: method,
-		Body:   bodyBytes,
+		Module:   module,
+		Method:   method,
+		Sequence: initialSeq, // needs to be nonzero
```

Comment:
We don't really want them to all have the same sequence number though. They should increment with each call within the session. I was thinking sequence number could be a member of the drpc.ClientConnection object. Then we could set it on the drpc.Call and increment for each SendMsg call, similar to what we're doing in drpc_call in the C code.